tests/unit/test___init__.py

```diff
@@ -31,6 +31,14 @@ def function_1():
         ),
         pytest.param(
             """
+def _function_1():
+    return
+""",
+            (f"2:0 {DOCSTR_MISSING_MSG}",),
+            id="private function docstring missing return",
```

Comment:
For this and yeilds could we add a test where the function is private, the docstring includes returns/ yields and the function does not return/ yield?

Reply:
and then maybe also a private function that does have returns/ yields and actually does return or yield and has the section in the docstring